### PR TITLE
🗽 Add script to convert haml templates to html

### PR DESCRIPTION
**Pull request by rowanseymour**
And use [djLint](https://djlint.com/) to format them

Note the sample conversions in this PR were done with https://github.com/nyaruka/django-hamlpy/pull/96 so it could do `"` wrapped attributes correctly.

### CODE REVIEW

File: templates/archives/archive_list.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{% extends "smartmin/list.html" %}
+{% load smartmin sms temba compress i18n humanize %}
+{% block title %}{{ title }}{% endblock %}
+{% block page-title %}{{ title }}{% endblock %}
+{% block content %}
```
Comment (rowanseymour) [inline templates/archives/archive_list.html:5]: you can tell djlint to leave blank lines alone.. which I did initially thinking this was weird looking... but then later convinced myself templates don't need them

Comment (ericnewcomer) [reply templates/archives/archive_list.html:5]: Wow, that's a lot of `%` chars. Django templates really are pretty ugly, but I'm still pretty convinced this is the right thing to do.

**Review by ericnewcomer — APPROVED** [review]

**Comment by ericnewcomer** [comment]
Is there an easy way to get named endblock tags?

**Comment by rowanseymour** [comment]
@ericnewcomer didn't know that was an option but could have hamlpy spit those out

**Comment by rowanseymour** [comment]
@ericnewcomer updated with endblock names.. I think the way forward here is selective incremental conversion so we minimize risk of breaking everything, and creating horrendous conflicts on open PRs

**Comment by ericnewcomer** [comment]
It'd be really nice to make sure we produce djlint qualified html. I noticed it likes spaces with variables for example, `{{ my_var }}` instead of `{{my_var}}`.

**Comment by rowanseymour** [comment]
Yeah that's why I made the change to properly support `""` for attributes - it was upset (I guess like black) about using `''`

**Review by norkans7 — APPROVED** [review]

**Comment by rowanseymour** [comment]
Last thing to be salvaged from this PR is the chart view...

**Comment by ericnewcomer** [comment]
Extracted that here: https://github.com/nyaruka/rapidpro/pull/4678

**Pull request closed without merging**